lib/Steps/Integrations/ReactNative.ts

```diff
@@ -231,9 +231,9 @@ export class ReactNative extends MobileProject {
     for (const script of buildScripts) {
       if (
         !script.shellScript.match(
-          /(packager|scripts)\/react-native-xcode\.sh\b/,
+          /\/scripts\/react-native-xcode\.sh/i,
```

Comment:
Why do we need to be case-insensitive in all checks?

Reply:
If not, we can just use `shellScript.includes('/scripts/react-native-xcode.sh')`